xcdat/bounds.py

```diff
@@ -140,6 +140,12 @@ def add_missing_bounds(self, width: float = 0.5) -> xr.Dataset:
             except KeyError:
                 pass
 
+            # determine if the axis is also a dimension by determining
+            # if there is overlap between the CF axis names and the dimension
+            # names. If not, skip over axis for validation.
+            if len(set(CF_NAME_MAP[axis]) & set(self._dataset.dims.keys())) == 0:
+                continue
+
```

Comment:
Just to make it clear what is going on...

For an axis like `lat', `set(CF_NAME_MAP[axis])` yields `{"Y", "latitude", "lat"}`. 

`set(self._dataset.dims.keys()))` yields `{'axis_nbounds', 'lat', 'lon', 'time'} `. 

`set(...) & set(...)` determines the intersection between these two groups. If there are elements in the resulting set, it means that the axis is also a dimension. If there are no elements in the set, then the axis is not a dimension. 

If the axis is not a dimension, there is no need to find bounds and we skip adding bounds.